manifest.webapp

```diff
@@ -24,6 +24,11 @@
       "type": "io.cozy.apps",
       "verbs": ["GET", "POST", "PUT"]
     },
+    "settings": {
+      "description": "Required by the cozy-bar to know which applications are coming soon",
```

Comment:
You're right, but I am too lazy to update my last 8 PRs :D.
We will update next time, and I am pretty sur that we will remove this permission before needing new infos from settings.